plugins/modules/vmware_vmkernel.py

```diff
@@ -792,7 +787,7 @@ def find_dvspg_by_key(self, dv_switch, portgroup_key):
 
         return None
 
-    def set_vsan_service_type(self):
+    def set_vsan_service_type(self, enable_vsan=True):
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Thanks for the notification, I just added a fragment.